doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst

```diff
@@ -0,0 +1,85 @@
+
+================================
+CORTX on Open Virtual Appliance
+================================
+An Open Virtual Appliance (OVA) is a Virtual Machine (VM) image that consists of a pre-installed and pre-configured operating system as well as one or more applications packaged for easy deployment and testing.  This document describes how to use a CORTX OVA for the purposes of single-node CORTX testing. 
+For our Japanese community, this document has been translated and is available `here <https://qiita.com/Taroi_Japanista/items/0ac03f55dce3f7433adf>`_.
+
+***********************
+Recommended Hypervisors
+***********************
+All of the following hypervisors should work: `VMware ESX Server <https://www.vmware.com/products/esxi-and-esx.html>`_,
+`VMware vSphere <https://www.vmware.com/products/vsphere.html>`_,
+`VMware Workstation <https://www.vmware.com/products/workstation-pro.html>`_,
+`VMware Fusion <https://www.vmware.com/in/products/fusion/fusion-evaluation.html>`_
+
+**Important**: If you are running the VM in any of the VMWare hypervisors, it is not recommended to use VMware Tools, as CORTX may break due to kernel dependencies. For the same reason, please do not update the operating system in the image as that also might cause it to fail.
+
+**Specifications:**
+
+- CORTX OVA image is created with following minimum configuration:
+
+  - RAM: 10GB
+  - Processor: 8
+  - NIC: 1
+  - OS Disk: 1 disk of 20GB
+  - Data Disks: 3 disks of 5GB each
+
+**Prerequisites:**
+
+- Download the `CORTX OVA <https://cortxova.s3.us-west-2.amazonaws.com/ova-2.0.0-713.ova>`_ from `our release page <https://github.com/Seagate/cortx/releases/latest>`_.
+- To perform the S3 IO operations,refer to these instructions in `S3 IO Operations <https://github.com/Seagate/cortx/blob/main/doc/ova/2.0.0/PI-6/S3_IO_Operations.md>`_.
```

Comment:
@venkatesh, Update the correct URL for S3 IO operations as the link is broken

Reply:
@mukul-seagate11 What is the correct URL? I believe this link will work once PR is merged.

Reply:
I see as its a newly created file so will be accessible once PR is merged so you can ignore